### PR TITLE
Adjust the `<Switch/>` to eliminate unnecessary space between elements when a label is not present

### PR DESCRIPTION
**Pull request by cmarin001**
In the present pull request, we have fine-tuned the spacing for the <Switch/> component, specifically for those scenarios when it is used in conjunction with the 'label' property. This adjustment will enhance its adaptability to centralize as a standalone element, even when there is no requirement for a 'label' accompanying the switch.

Screenshot from 2023-06-01 12-32-04

### CODE REVIEW

**Comment by Andresbl123** [comment]
Cumple con los requerimientos

File: src/components/inputs/Switch/index.jsx [hunk 1]
```diff
@@ -23,13 +23,15 @@ const Switch = (props) => {
   } = props;
 
   const transformedSize = sizes.includes(size) ? size : defaultSize;
+  const tranformedGap = label ? "10px" : "0px";
+  const transformedAlignment = label ? "space-between" : "center";
```
Comment (wfercanas) [inline src/components/inputs/Switch/index.jsx:27]: Es preferible que lo nombre transformedJustify, porque en este contexto align se puede leer como relacionado a align-items.

**Review by wfercanas — CHANGES_REQUESTED** [review]

File: src/components/inputs/Switch/index.jsx (hunk 1 above)
Comment (cmarin001) [reply src/components/inputs/Switch/index.jsx]: resuelto

**Comment by Andresbl123** [comment]
Ajustes realizados

**Review by wfercanas — APPROVED** [review]

**Pull request merged**